src/fdk_rdf_parser/classes/cpsvno_service.py

```diff
@@ -54,3 +58,6 @@ class Service:
     subject: Optional[List[str]] = None
     homepage: Optional[List[str]] = None
     dctType: Optional[List[ReferenceDataCode]] = None
+    thematicareas: Optional[List[EuDataTheme]] = None
```

Comment:
```suggestion
    thematicAreas: Optional[List[str]] = None
```

Reply:
thematicArea -> uri til alle tema
losThemes -> liste med fullstendige los-tema
euDataThemes -> liste med fullstendige eu-tema

Reply:
Tror dette krever en refaktorering av `split_themes` og dermed en refaktorering av Datasett-parsingen. Alternativet er at jeg lager en egen `split_themes`-funksjon til `Service`-parsingen men er ikke veldig for det.

Reply:
Lag en ny generisk funksjon, i feks utils-fila, som du nå tar i bruk for tjenester og lag et issue om å refaktorere datasett til å bruke den nye generiske funksjonen

Reply:
Jepp, god idé! Ny commit kommer etter lunsj^^